content/docs/get-started/install/migrating-2.0.md

```diff
@@ -113,7 +87,7 @@ Then run `go mod download`
 Update your package reference to the latest version of the SDK:
 
 ```csharp
-<PackageReference Include="Pulumi" Version="2.0.0-beta.2" />
+<PackageReference Include="Pulumi" Version="2.0.0" />
```

Comment:
so this will pin them to 2.0.0 - don't we want to use `2.*` as it will grown as the series grows?

Reply:
Ah, yes, of course. Let me fix that.